spec/migrations/20150806190349_rename_columns_starting_with_miq_event_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'spec_helper'
+require Rails.root.join('db/migrate/20150806190349_rename_columns_starting_with_miq_event')
+
+describe RenameColumnsStartingWithMiqEvent do
```

Comment:
This test is not necessary because it is just testing ActiveRecord::Migration, which we don't need to do.